source/client-side-encryption/client-side-encryption.rst

```diff
@@ -294,6 +325,7 @@ MongoClient Changes
       bypassAutoEncryption: Optional<Boolean>; // Default false.
       extraOptions: Optional<Map<String, Value>>;
       tlsOptions: Optional<Map<String, TLSOptions>>; // Maps KMS provider to TLS options.
+      encryptedFieldConfigMap: Map<String, Map<String, Document>>; // Maps namespace to EncryptedFieldConfig.
```

Comment:
I think it is OK to leave as-is. The type is expected to be the same as `schemaMap`.

Reply:
Could the type be the same as schemaMap: `Optional<Map<String, Document>>`?  It seems similarly optional, and the generic type representing EncryptedFieldConfig could just be `Document` rather than `Map<String, Document>`.  I'm not sure what purpose the extra level of `Map` is serving.